dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -49,6 +49,8 @@ repertiore.LOW=Cipher suites without encryption
 repertiore.CUSTOM=Custom list of cipher suites
 repertoire.verifyHostname=Host name verification
 repertoire.verifyHostname.desc=Specifies whether host name verification for outbound connections using a specific SSL configuration is enabled. If set to true, then all outbound SSL connections that use the specified SSL configuration undergo verification of the target server host name against that server's certificate. The attribute is set to false by default.
+repertoire.trustCacerts=Use the cacerts file for trust
+repertoire.trustCacerts.desc=Specifies whether to use the JDK cacerts file to establish trust or not.  If set to true, then the JDK cacerts file will be in addition to the configured trust file to establish trust.  The attribute is set to false by default.
```

Comment:
Change
"Specifies whether to use the JDK cacerts file to establish trust or not.  If set to true, then the JDK cacerts file will be in addition to the configured trust file to establish trust.  The attribute is set to false by default."
To
"Specifies whether to use the JDK cacerts file to establish trust. If set to true, then the JDK cacerts file is used to establish trust, in addition to the configured trust file. The attribute is set to false by default."